src/particle_data.cpp

```diff
@@ -51,6 +52,9 @@ ParticleData::ParticleData()
   // Create microscopic cross section caches
   neutron_xs_.resize(data::nuclides.size());
   photon_xs_.resize(data::elements.size());
+
+  // pht_storage_.resize(model::cells.size(), 0.0);
+  pht_storage_.resize(model::cells.size(), 0.0);
```

Comment:
Should remove the extraneous comment here.

Reply:
yes!